tests/ieee802154_hal/Makefile

```diff
@@ -0,0 +1,41 @@
+include ../Makefile.tests_common
+
+DISABLE_MODULE += auto_init_at86rf2xx auto_init_nrf802154
+
+USEMODULE += od
+USEMODULE += luid
+USEMODULE += ieee802154
+USEMODULE += shell
+USEMODULE += ps
+USEMODULE += event_thread_highest
+USEMODULE += event_callback
+USEMODULE += xtimer
+USEMODULE += ieee802154_radio_hal
+
+CFLAGS += -DEVENT_THREAD_HIGHEST_STACKSIZE=1024
+
+NETDEV ?= 0
+
+CFLAGS += -DNETDEV=$(NETDEV)
+
+# define the driver to be used for selected boards
+ifneq (,$(filter samr21-xpro,$(BOARD)))
+  DRIVER := at86rf233
+endif
+ifneq (,$(filter iotlab-m3 fox,$(BOARD)))
+  DRIVER := at86rf231
+endif
+ifneq (,$(filter nrf52840dk,$(BOARD)))
+  DRIVER := nrf802154
+endif
+ifneq (,$(filter cc2538dk,$(BOARD)))
```

Comment:
Would be fine to check for the CC2538 MCU here. However, you were talking about the remote-rev% board somewhere, not cc2538dk.